examples/simple_example/main.py

```diff
@@ -0,0 +1,3 @@
+from simple_example.__main__ import main
+
```

Comment:
What's this file for?

Reply:
For the CI test to trigger the different examples